examples/dapperish.go

```diff
@@ -78,7 +79,7 @@ func server() {
 }
 
 func main() {
-	opentracing.InitDefaultTracer(NewTracer("dapperish_tester"))
+	opentracing.InitDefaultTracer(dapperish.NewTracer("dapperish_tester"))
```

Comment:
moved one level up to keep it in `main` package, while the actual dapperish implementation is in `dapperish` subpackage